src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,9 +35,11 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
-        cout<<"\t--robot                <robot-name>: robot name"<<endl;
+        cout<<"\t--namePrefix             <name-prefix>: prefix of the yarprobotstatepublisher ports"<<endl;
+        cout<<"\t--tfPrefix               <tf-prefix>: prefix of the published TFs"<<endl;
```

Comment:
An example of how these prefixes are used in constructing the output port names or relevant output quantity names would be useful for the user to understand more easily the setting up of the robotstatepublisher.